pkg/service/restore/service_restore_integration_test.go

```diff
@@ -1001,21 +1002,49 @@ func restoreWithVersions(t *testing.T, target Target, keyspace string, loadCnt,
 		t.Fatal(err)
 	}
 
+	firstCorruptCrc32 := make(map[string]struct{})
+	for _, f := range firstCorrupt {
+		firstCorruptCrc32[sstable.Crc32FileNameFromGivenSSTableFile(f)] = struct{}{}
+	}
+	bothCorruptCrc32 := make(map[string]struct{})
+	for _, f := range bothCorrupt {
+		bothCorruptCrc32[sstable.Crc32FileNameFromGivenSSTableFile(f)] = struct{}{}
+	}
+	secondCorruptCrc32 := make(map[string]struct{})
+	for _, f := range secondCorrupt {
+		secondCorruptCrc32[sstable.Crc32FileNameFromGivenSSTableFile(f)] = struct{}{}
+	}
+	for f := range bothCorruptCrc32 {
+		firstCorruptCrc32[f] = struct{}{}
+		secondCorruptCrc32[f] = struct{}{}
+	}
```

Comment:
Can't we simply append .crc32 files above in:
```go
switch {
			case len(firstCorrupt) < corruptCnt:
				firstCorrupt = append(firstCorrupt, item.Path, sstable.Crc32FileNameFromGivenSSTableFile(item.Path))
			case len(bothCorrupt) < corruptCnt:
				bothCorrupt = append(bothCorrupt, item.Path, ...)
			case len(secondCorrupt) < corruptCnt:
				secondCorrupt = append(secondCorrupt, item.Path, ...)
			}
```
without creating new maps/slices that are appended afterwards?

Reply:
But you have counters there. Didn't want to change the exisitng logic.